src/scss/components/_icons.scss

```
@@ -9,10 +9,23 @@
   font-display: swap;
 }
 
+.circle-icon {
+  background: $primary;
+  padding: 20px;
+  border-radius: 100%;
+}
```

```suggestion
.circle-icon {
  padding: 20px;
  border-radius: 100%;
}
```

Utiliser classes utilitaires pour gérer les couleurs de fond : https://getbootstrap.com/docs/5.1/utilities/background/